src/container.ts

```diff
@@ -29,6 +36,23 @@ const container = awilix.createContainer({
   injectionMode: awilix.InjectionMode.CLASSIC
 });
 
+// Server port.
+const DEFAULT_SERVER_PORT = "80";
+const serverPort: number = parseInt(
+  process.env.PORT || DEFAULT_SERVER_PORT,
+  10
+);
+container.register({
+  serverPort: awilix.asValue(serverPort)
+});
+
+// Server environment.
+const DEFAULT_ENVIRONMENT = EnvironmentNodeEnvEnum.DEVELOPMENT;
```

Comment:
default environment should be production, in case you forget to set the proper variable during a deploy... by default it's (usually) better to have a production logic running on a dev machine than a dev logic running in production...